gr-data.csv

```diff
@@ -342,7 +342,7 @@ https://github.com/johnrengelman/shadow,d8117e942b7d98e3274cb1cc9dbb4e4e978a2a58
 https://github.com/johnrengelman/shadow,d8117e942b7d98e3274cb1cc9dbb4e4e978a2a58,.,"com.github.jengelman.gradle.plugins.shadow.transformers.ServiceFileTransformerSpec.""non-excluded path META-INF/services/io.dropwizard.logging.AppenderFactory can be transformed""",ID-HtF,,,
 https://github.com/johnrengelman/shadow,d8117e942b7d98e3274cb1cc9dbb4e4e978a2a58,.,"com.github.jengelman.gradle.plugins.shadow.transformers.ServiceFileTransformerSpec.""transforms service file [path: META-INF/services/com.acme.Bar, input1: foo bar, input2: zoo, output: foo bar zoo]""",ID-HtF,,,
 https://github.com/johnrengelman/shadow,d8117e942b7d98e3274cb1cc9dbb4e4e978a2a58,.,"com.github.jengelman.gradle.plugins.shadow.transformers.ServiceFileTransformerSpec.""transforms service file [path: META-INF/services/com.acme.Foo, input1: foo, input2: bar, output: foo bar]""",ID-HtF,,,
-https://github.com/KelinTan/spring-boot-archetype,e4282acc9d4f94d0e9be44301222ecf3a5e29c5f,archetype-common,com.kelin.archetype.common.json.JsonConverterTest.testSerialize,ID,,,
+https://github.com/KelinTan/spring-boot-archetype,e4282acc9d4f94d0e9be44301222ecf3a5e29c5f,archetype-common,com.kelin.archetype.common.json.JsonConverterTest.testSerialize,ID,Opened,https://github.com/KelinTan/spring-boot-archetype/pull/135,
```

Comment:
https://github.com/KelinTan/spring-boot-archetype/pull/135/files#diff-3d38f63525ddfb9e94473244afe87fc826da97199c5e387d776401408b492501R26 uses `==` (rather than `equals`) to compare strings.

Reply:
Yes, thanks for that note. however, the PR got merged by the developer. Can i go ahead and update it to "Accepted"?

Reply:
Yes, you can change to `Accepted`.

What is the difference between different equal operators in Kotlin? How do they compare to `==` and `equals` in Java?

Reply:
Thanks! Scala did the same with `==`, but it seems developers prefer to write longer code. :) We'll see what the future holds for Kotlin.